src/applications/accredited-representative-portal/accreditation/21a/pages/helpers/backgroundInformationDetailsPageSchema.js

```diff
@@ -43,6 +43,10 @@ const backgroundInformationDetails = ({
           name: `${path}-file-input`,
           fileUploadUrl: url,
         }),
+        warnings: {
+          'ui:field': 'BasicArrayField',
+          'ui:title': 'Conviction document upload warnings',
```

Comment:
I think this will need to be dynamic string, as not every page that uses this schema abstract is conviction document

Reply:
Oh whoops, I see now. I'll update this @tjheffner

Reply:
@tjheffner 
Any thoughts on dynamic title versus stripping context prefix? I don't know the obvious way of choosing / standard. Stripped context version would be:
```javascript
'ui:title': 'Document upload warnings',
```

Reply:
I think that is totally acceptable in this case, good call :+1: